app/views/markdowns/explanation.haml

```diff
@@ -0,0 +1,19 @@
+%h1 Markdown explanation
+%p Markdown is used in question rendering and here is simple guide to get you started.
+%ul
+	%li 
+		:markdown 
```

Comment:
I'm not sure the `:markdown` filter nets you too much here over just writing the markup in HAML, but if we keep it, I recommend moving it to the parent of the `<ul>`. e.g.

```haml
:markdown
  * Surround text with \*asterisks\* or \_underscores\_for *italic text*.
  * Surround text with \*\*double asterisks\*\* or \_\_double underscores\_\_
    for **bold text**.
  * etc.
```